Formula/swiftgen.rb

```diff
@@ -2,18 +2,17 @@ class Swiftgen < Formula
   desc "Swift code generator for assets, storyboards, Localizable.strings, …"
   homepage "https://github.com/SwiftGen/SwiftGen"
   url "https://github.com/SwiftGen/SwiftGen.git",
-      :tag      => "6.1.0",
-      :revision => "c6477a6950d313c94c964c6c8ec960e3aeccb8f6"
-  head "https://github.com/SwiftGen/SwiftGen.git"
+      :tag      => "6.2.0",
+      :revision => "c62c7ff1c86323b9c1e4c12bccfca19737dd2e56"
+  revision 1
```

Comment:
no need to have a revision for the new release.

Reply:
The revision was needed because during my rebase CI failed for 10.13 and 10.14 while trying to create the bottles, with "mkdir bottle: directory already exists". I'm assuming because it previously created the dir from previous runs of the CI when it became green before rebase.
That's why I had to force a rebuild of the bottles using `revision 1`

Reply:
That was probably a CI hickup unrelated to your changes.